internal/servers/generic_server.go

```
@@ -41,42 +41,44 @@ import (
-	pathCompiler     *masks.PathCompiler[O]
-	pathCache        map[string]*masks.Path[O]
-	pathCacheLock    *sync.Mutex
+	logger               *slog.Logger
+	service              string
+	dao                  *dao.GenericDAO[O]
+	attributionLogic     auth.AttributionLogic
+	tenancyLogic         auth.TenancyLogic
+	disallowSharedTenant bool
+	template             proto.Message
+	metadataField     protoreflect.FieldDescriptor
+	nameField         protoreflect.FieldDescriptor
```

golangci-lint (2.12.2)

[error] field nameField is unused

(unused)